rpc/client.go

```diff
@@ -76,12 +76,12 @@ type internalConn struct {
 
 // Client is a Cockroach-specific RPC client.
 type Client struct {
-	key       string // cache key for later removal from cache
-	addr      util.UnresolvedAddr
-	Closed    chan struct{}
-	conn      unsafe.Pointer // holds a `internalConn`
-	healthy   atomic.Value   // holds a `chan struct{}` exposed in `Healthy`
-	tlsConfig *tls.Config
+	key            string // cache key for later removal from cache
+	addr           util.UnresolvedAddr
+	Closer, Closed chan struct{}
```

Comment:
I don't think Closer needs to be exported, does it?  It seems like calling "Close()" is the preferred method here.